Magento 86

suraiyavic opened the pull request.

suraiyavic committed:
[MAG-143] Using Plugin to Refresh Tax when `Store Front` Currency Selection Changes

suraiyavic committed:
[MAG-123] Added Default Country of Origin on UI

suraiyavic committed:
[MAG-113] Pulling Proper Country of Origin for Using in DHL API Call

suraiyavic committed:
[MAG-86] Added Plugin to Remove a UI element from Magento Checkout Page

lizzyWithReach commented on Plugin/Magento/Checkout/Block/Checkout/LayoutProcessor.php:
```diff
@@ -0,0 +1,23 @@
+     */
+    public function aroundProcess($subject, $proceed, $jsLayout)
+    {
+        unset($jsLayout['components']['checkout']['children']['sidebar']['children']['summary']['children']['totals']['children']['grand-total']['config']['basicCurrencyMessage']);
```
this is crazy haha thank you for the references in the ticket, that was very helpful. Do you think it would make sense to link to Magento official documentation about this?

although maybe people who work with M2 are used to this, or future devs could just look up the ticket..

suraiyavic replied:
commit 7773de5ad3c6d1d0c4894180cf3b1d2f9e007757
Thanks!

lizzyWithReach reviewed (approved):
another comment q, approving because it's not a big enough point to stop this from going out. Thank you!

suraiyavic committed:
[MAG-143] Fixed a Typo

suraiyavic committed:
Merge branch 'magento_143' into magento_123

suraiyavic committed:
[MAG-123] Making showInStore Consistent for `Default HS Code` and `Default HS Code Country`

suraiyavic committed:
Merge branch 'magento_123' into magento_113

suraiyavic committed:
[MAG-113] Updated Comments

suraiyavic committed:
Merge branch 'magento_113' into magento_86

suraiyavic committed:
[MAG-86] Added Helpful Hints

Copied a helpful hint/link from the JIRA ticket MAG-86 and pasted in the source
 code so that developers would know why the work was done in that particular way.

lizzyWithReach reviewed (approved):
Perfect, thank you!
